Internet data/fetch_links.py

```diff
@@ -0,0 +1,29 @@
+import urllib.request
+from bs4 import BeautifulSoup
+# Test case:
+# https://flyingmedusa.github.io/simple_page/index.html
+
+
+def extract_ol(url):
+    page = urllib.request.urlopen(url)
+    html = BeautifulSoup(page.read(), 'html.parser')
+    ol_nodes = html.find_all('ol')
+    children = []
+    for ol in ol_nodes:
+        children.append(ol.findChildren())
+    return children
+
+
+def main():
+    url = input("Please paste the url\t")
+
+    ols_children = extract_ol(url)
+
+    for li_list in ols_children:
+        print()
```

Comment:
why do you need an empty `print()` statement here?

Reply:
Just for the esthetics. When a page has more than one ordered list, this empty print() separates lists. I believe it is more user-friendly now.